lib/js-file.js

```diff
@@ -447,17 +440,18 @@ JsFile.prototype = {
      * @param {Function} cb
      */
     iterateTokensByType: function(type, cb) {
-        var types = (typeof type === 'string') ? [type] : type;
-        var typeIndex = {};
-        types.forEach(function(type) {
-            typeIndex[type] = true;
-        });
-
-        this.getTokens().forEach(function(token, index, tokens) {
-            if (typeIndex[token.type]) {
-                cb(token, index, tokens);
+        var tokens;
+        if (Array.isArray(type)) {
```

Comment:
It seems argument check should look like the same way as in `getNodesByType` method